public/app/features/dashboard-scene/edit-pane/useDropAndPaste.ts

```diff
@@ -36,33 +53,77 @@ export function useDropAndPaste() {
       if (clipboardData.types.includes('text/plain')) {
         // Handle plaintext paste
         const text = clipboardData.getData('text/plain');
-        alert(`Pasted text: \n${text}`);
+        notify.success(`Pasted text: \n${text}`);
 
         return;
       }
 
       if (clipboardData.types.includes('text/html')) {
         // Handle HTML paste
         const html = clipboardData.getData('text/html');
-        alert(`Pasted HTML:\n${html}`);
+        notify.success(`Pasted HTML:\n${html}`);
+
+        return;
       }
 
-      if (clipboardData.types.includes('image/png') || clipboardData.types.includes('image/jpeg')) {
```

Comment:
Only `text/plain`, `text/html`, and `image/png` are [properly supported](https://www.w3.org/TR/clipboard-apis/#mandatory-data-types-x) in clipboard events (although you can technically create a payload with `application/json`)